sdks/python/apache_beam/runners/interactive/extensions/interactive_beam_side_panel/interactive_beam_side_panel/_version.py

```diff
@@ -0,0 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+version_info = (0, 1, 0)
```

Comment:
should the version match beam version? We do not have other sub-components with their own versions.

Reply:
This is also not part of the Python SDK. If this looks weird, could we move the `extensions` directory out of `sdks/python`?